addons/crm/models/crm_team.py

```diff
@@ -723,6 +693,24 @@ def action_opportunity_forecast(self):
         action = self.env['ir.actions.actions']._for_xml_id('crm.crm_lead_action_forecast')
         return self._action_update_to_pipeline(action)
 
+    def action_open_leads(self):
+        action = self.env['ir.actions.actions']._for_xml_id('crm.crm_case_form_view_salesteams_opportunity')
+        rcontext = {
+            'team': self,
+        }
+        action['help'] = self.env['ir.ui.view']._render_template('crm.crm_action_helper', values=rcontext)
+        return action
+
+    def action_open_unassigned_leads(self):
+        action = self.action_open_leads()
+        context_str = action.get('context', '{}')
+        if context_str:
+            context = safe_eval(action['context'], {'active_id': self.id, 'uid': self.env.uid})
```

Comment:
Shouldn't we encapsulate in a try / except in case people did strange things with their context ? And just have a simple / standard context if we can't evaluate for some reason.